docs/samples/shards/General/Match/3.edn

```diff
@@ -1,10 +1,9 @@
-;; No match + no nil case + passthrough
-false
-(Match
- :Cases
- [true (-> (Msg "Not a match")) ;; no match
-  true (-> (Msg "Not a match either"))] ;; no match
-            ;; reached end of cases with no match found => no shards activated
- :Passthrough
- true) ;; input allowed to passthrough to Match shard's output
-(Log) ;; print Match shard's output => "false"
+;; multiple matches + nil match at start + with passthrough
+1
+(Match [
+        nil (-> "Matched nil")     ;; case processed, match found
+        1 (-> "Matched 1")         ;; case ignored
+        (+ 0 1) (-> "Matched 1a")  ;; case ignored
+        2 (-> "Matched 2")]        ;; case ignored
```

Comment:
For the sample with `:Passthrough true` I think it would be good to log/msg on each case so that the console output also shows which case was matched.

Something like:
```suggestion
        nil (-> (Msg "Matched nil"))     ;; case processed, match found
        1 (-> (Msg "Matched 1"))         ;; case ignored
        (+ 0 1) (-> (Msg "Matched 1a"))  ;; case ignored
        2 (-> (Msg "Matched 2"))]        ;; case ignored
```

Reply:
Yes, that's a good idea. Fixed (with fixup, rebase, force-push).